pkg/converter/convert_unix.go

```diff
@@ -771,6 +783,31 @@ func LayerConvertFunc(opt PackOption) converter.ConvertFunc {
 			return nil, nil
 		}
 
+		// Use remote cache to avoid duplicate conversion
+		info, err := cs.Info(ctx, desc.Digest)
+		if err != nil {
+			return nil, errors.Wrapf(err, "get blob info %s", desc.Digest)
+		}
+		if info.Labels[LayerAnnotationDestDigest] != "" {
+			destInfo, err := cs.Info(ctx, digest.Digest(info.Labels[LayerAnnotationDestDigest]))
+			if err != nil {
+				return nil, errors.Wrapf(err, "get blob info %s", desc.Digest)
+			} else {
+				destDesc := ocispec.Descriptor{
+					Digest:    destInfo.Digest,
+					Size:      destInfo.Size,
+					MediaType: MediaTypeNydusBlob,
+					Annotations: map[string]string{
+						// Here we do not add annotation containerd.io/snapshot/nydus-source-digest
+						LayerAnnotationUncompressed:               destInfo.Digest.String(),
+						LayerAnnotationNydusBlob:                  "true",
+						LayerAnnotationFromCache: 				   "true",
```

Comment:
Ditto, does this code has been formatted?

Reply:
It is formatted locally, too.May need to push again.

Reply:
fixed